curriculum/challenges/english/blocks/workshop-bookstore-page/68ebdbacdd3fa474132cc975.md

```diff
@@ -0,0 +1,46 @@
+---
+id: 68ebdbacdd3fa474132cc975
+title: Step 2
+challengeType: 0
+dashedName: step-2
+---
+
+# --description--
+
+Below the `<h1>` element, add a `<p>` element with this text: `Browse our collection of amazing books!`
+
+# --hints--
+
+Your `p` element should have an opening tag. Opening tags have this syntax: `<elementName>`.
+
+```js
+assert.exists(document.querySelector('p'));
+```
+
+Your `p` element should have a closing tag. Closing tags have a `/` and the same element name: `</p>`.
+
+```js
+assert.match(code, /<\/p\s*>/);
+```
```

Comment:
see my note about earlier on why these tests are not needed. 
we just need one test to make sure the p element exists